### DISCUSSION TITLE
is there python api support?

### DISCUSSION THREAD

**Discussion by SarahPeterson2854** [sandboxie-plus/Sandboxie#2370] [Various ideas and suggestions]
is there python api support?

There is a library from 2012 but it's deprecated, throws an error trying to parse the ini

**Comment by bastik-1001** [comment]
I am not sure if [that](https://sandboxie-plus.github.io/sandboxie-docs/Content/SBIEDLLAPI.html) is related, but apparently the dll provides an API to be used by other applications.
